ui/src/utils/prop-types.js

```diff
@@ -255,6 +255,8 @@ export const awsProvisionerWorkerType = shape({
 
 export const WorkerManagerWorkerPoolSummary = shape({
   workerPoolId: string,
+  workerPoolId1: string,
+  workerPoolId2: string,
```

Comment:
`workerPoolId1`, `workerPoolId2` are not defined in the schema so we should probably remove them. This is a similar thing as https://github.com/taskcluster/taskcluster/pull/843#discussion_r291588795.

Reply:
ah, yes. Missed a spot